pkg/models/vendor_verify_info.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2018 The OpenPitrix Authors. All rights reserved.
+// Use of this source code is governed by a Apache license
+// that can be found in the LICENSE file.
+
+package models
+
+import (
+	"context"
+	"time"
+
+	"openpitrix.io/openpitrix/pkg/logger"
+	"openpitrix.io/openpitrix/pkg/pb"
+	"openpitrix.io/openpitrix/pkg/util/pbutil"
+)
+
+type Vendor struct {
+	UserId             string
+	CompanyName        string
+	CompanyWebsite     string
+	CompanyProfile     string
+	AuthorizerName     string
+	AuthorizerEmail    string
+	AuthorizerPosition string
+	AuthorizerPhone    string
+	BankName           string
+	BankAccountName    string
+	BankAccountNumber  string
+	Status             string
+	RejectMessage      string
+	SubmitTime         time.Time
+	StatusTime         time.Time
+	UpdateTime         time.Time
+}
+
+func (vendor *Vendor) ParseInputParams2Obj(in *pb.SubmitVendorVerifyInfoRequest) *Vendor {
```

Comment:
This code cannot be reused and no need to be placed here. I recommend put it into `SubmitVendorVerifyInfo`